src/orgwarden/__main__.py

```diff
@@ -1,17 +1,63 @@
+import sys
+from typing import Annotated
 import typer
-from .repo_crawler import fetch_org_repos
+from orgwarden.audit import audit_repository
+from orgwarden.repository import Repository
+from orgwarden.repo_crawler import fetch_org_repos
 
 app = typer.Typer()
 
 
 @app.command()
-def main(org_name: str) -> None:
+def ls_repos(org_name: str) -> None:
+    """
+    Lists all public, non-forked repositories for the specified <org_name>
+    """
     repos = fetch_org_repos(org_name)
 
     print(f"~~~~~ Public repositories found for {org_name} ~~~~~")
     for repo in repos:
         print(f"{repo.org}/{repo.name} - {repo.url}")
 
 
+@app.command()
+def audit_repo(
```

Comment:
This will work for organizations on GitHub, but will not work for internal organizations (such as [https://github.gatech.edu/sse-center).](https://github.gatech.edu/sse-center).) Can you update the function signature to take a URL as input and then parse the repo owner and repo name?

If you do this, you can also name this function `audit` and handle both the `audit_repo` and `audit_org` scenarios by detecting if the URL points to an org or a repo.